cypress/integration/ColorPicker.spec.ts

```diff
@@ -1,15 +1,11 @@
 import * as h from '../helpers';
 
 const getColorInput = () => cy.get('[type="text"]');
-
 const getColorPickerPopup = () => cy.get('[role=dialog]');
-
-const expandHex = (hex: string) => {
-  const shorthandRegex = /^#?([a-f\d])([a-f\d])([a-f\d])$/i;
-  return hex.replace(shorthandRegex, function(m: string, r: string, g: string, b: string) {
-    return r + r + g + g + b + b;
-  });
-};
+const getOpenButton = () => cy.get('[data-testid=open]');
+const getResetButton = () => cy.contains('button', 'Reset');
+const getSubmitButton = () => cy.get('[aria-label=Submit]');
+const getSwatch = (color: string) => cy.get(`div[color="${color}"]`);
```

Comment:
Still thinking about this one 🤔 

It is kind of brittle because we don't technically need to pass `color` as a prop to the styled `Container` element ([see code](https://github.com/lychyi/canvas-kit/blob/remove-data-attr/modules/color-picker/react/lib/parts/ColorSwatch.tsx#L14-L47)) but we do today so this selector works "as is".

However, one thought that came to mind is that these swatches aren't exactly accessible. The swatches should probably have an `aria-label` attr with the color name or the hex value. This would also solve the selector problem.

However, if we're going to go down that rabbit hole, it'd probably be more prudent to make another PR to address a11y in one shot.

Reply:
Agreed